src/Data/CartItem/CartItemDataClass.ts

```diff
@@ -21,11 +21,11 @@ async function attributes(): Promise<DataAttributeDefinitions> {
   }
 }
 
-export const CartItem = provideDataClass('CartItem', {
+export const CartItem = provideDataClass('CartItem', () => ({
```

Comment:
* [ ] FYI This is now more complicated than before. Is there a reason for that?

Reply:
This is needed, since `localStorageDataConnection` requires a instance ID to be configured. If you visit the multi-tenancy app without an instance ID the `CartItem` params would be executed right away. With making it a callback it will only be executed, if `CartItem` is actually needed.